do not hardcode -I/usr/include as it's useless and breaks cross-compiles

--- a/setup.py
+++ b/setup.py
@@ -370,7 +370,7 @@ kw = {'name':"pycrypto",
       'ext_modules': plat_ext + [
             # _fastmath (uses GNU mp library)
             Extension("Crypto.PublicKey._fastmath",
-                      include_dirs=['src/','/usr/include/'],
+                      include_dirs=['src/'],
                       libraries=['gmp'],
                       sources=["src/_fastmath.c"]),
 
